src/NumericalAlgorithms/Spectral/SwshDerivatives.cpp

```diff
@@ -82,7 +82,7 @@ void compute_derivative_coefficients(
         pre_derivative_modes->data().data() +
             i * size_of_libsharp_coefficient_vector(l_max),
         size_of_libsharp_coefficient_vector(l_max)};
-    output_mode_view = input_mode_view * modal_derivative_operator;
+    output_mode_view = modal_derivative_operator * input_mode_view;
```

Comment:
Is there a test that could be added that would've caught this?

Reply:
The operator * was chosen to be commutative for the diagonal modal operator types and modal vector types (because you could interpret the vector as row or column, and because noncommutative * should be introduced only if really necessary).
  So, this is a style change because @scheel preferred this order.

Reply:
Aha, okay. Thanks for explaining that :)